conf/modules/telemetry_xbee_api.xml

```diff
@@ -24,6 +25,7 @@
     <define name="DOWNLINK_DEVICE" value="$(MODEM_PORT_LOWER)"/>
     <define name="XBEE_UART" value="$(MODEM_PORT_LOWER)"/>
     <define name="XBEE_BAUD" value="$(MODEM_BAUD)"/>
+    <define name="XBEE_CHANNEL" value="$(MODEM_CHANNEL)"/>
```

Comment:
Why do you want to change it to a `configure`? This is not general for modems and currently only applicable to xbee's. My suggestion is to just remove this line, together with all the board file changes.